test/service/test_manifest.py

```diff
@@ -340,16 +436,28 @@ def test_manifest_zarr(self):
             with self.subTest(is_single_part=single_part):
                 with mock.patch.object(type(config), 'disable_multipart_manifests', single_part):
                     response = self._get_manifest(ManifestFormat.compact, filters)
-                    self.assertEqual(200, response.status_code, 'Unable to download manifest')
+                    self.assertEqual(200, response.status_code)
                     # Cannot use response.iter_lines() because of https://github.com/psf/requests/issues/3980
-                    lines = response.content.decode('utf-8').splitlines()
+                    lines = response.content.decode().splitlines()
                     tsv_file = csv.DictReader(lines, delimiter='\t')
                     rows = list(tsv_file)
-                    self.assertEqual(len(rows), 13)  # 12 related file, one original
-                    self.assertEqual(len(rows), len({row['file_uuid'] for row in rows}), 'Rows are not unique')
-                    # 2 pairs of files out of the 13 have matching checksums
-                    self.assertEqual(11, len({row['file_sha256'] for row in rows}), 'Incorrect unique sha256 count')
-                    self.assertEqual(11, len({row['file_crc32c'] for row in rows}), 'Incorrect unique crc32c count')
+                    rows = [dict(file_crc32c=row['file_crc32c'],
+                                 file_sha256=row['file_sha256'],
+                                 file_name=row['file_name'],
+                                 file_uuid=row['file_uuid'],
+                                 specimen_from_organism_organ=row['specimen_from_organism.organ']) for row in rows]
+                    self.assertEqual(expected, rows)
+
+                    response = self._get_manifest(ManifestFormat.curl, filters)
+                    self.assertEqual(200, response.status_code)
+                    lines = response.content.decode().splitlines()
+                    file_output = 'output="587d74b4-1075-4bbf-b96a-4d1ede0481b2/'
+                    output_paths = set()
+                    for line in lines:
+                        if line.startswith(file_output):
+                            _, _, file_name = line.partition(file_output)
```

Comment:
Partition performs a search. If you know the prefix, you know where to slice the string, eliminating the need for a search.